flowermd/library/systems.py

```diff
@@ -1,3 +1,37 @@
 """Examples for the Systems class."""
 
 # This is a placeholder for any class that inherits from base.system
+import numpy
+from scipy.spatial.distance import pdist
+
+
+class SingleChainSystem(System):
+    """Builds a box around a single chain.
+
+    Calculates the maximum distance of the chain using scipy.spatial.distance.pdist().
+
+    Parameters
+    ----------
+    See System class.
+
+    """
+
+	def __init__(self, molecules, base_units=dict()):
+		super(SingleChainSystem, self).__init__(
+			molecules=molecules,
+			base_units=base_units
+           	 )
+
+        def _build_system(self):
+            chain = self.all_molecules[0]
+            children_pos_array = np.zeros((len(chain.children),3))
```

Comment:
Instead of building a new array that has the position of all particles in the single chain, we can use mbuild's API.

`self.all_molecules[0].xyz` gives an array with all particle coordinates

Reply:
[Here is the function in mBuild's compound class](https://github.com/mosdef-hub/mbuild/blob/d79ac89dbd04e9891779a59c5c44355a9e11be23/mbuild/compound.py#L1340) so you can see how it differs from [the pos() function](https://github.com/mosdef-hub/mbuild/blob/d79ac89dbd04e9891779a59c5c44355a9e11be23/mbuild/compound.py#L1266)